logging_windows.go

```diff
@@ -13,7 +13,7 @@ import (
 	"os"
 	"sync"
 
-	"code.google.com/p/winsvc/eventlog"
+    "golang.org/x/sys/windows/svc/eventlog"
```

Comment:
it looks like we have some indenting issues here -- could you run gofmt on your files changed, just to make sure we stay consistent with indenting and so on?